tasks/sign-binaries/sign-binaries.yaml

```diff
@@ -42,36 +45,203 @@ spec:
+        echo -n "$mac_digest" > "$(results.unsignedMacDigest.path)"
+
+        echo "Pushing unsigned Windows content to $(params.quayURL)..."
+        output=$(oras push "$(params.quayURL)" "$WINDOWS_CONTENT")
         windows_digest=$(echo "$output" | grep 'Digest:' | awk '{print $2}')
         echo "Digest for windows content: $windows_digest"
         echo -n "$windows_digest" > "$(results.unsignedWindowsDigest.path)"
+    - name: sign-mac-binaries
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      volumeMounts:
+        - name: mac-ssh-key-vol
+          mountPath: "/etc/secrets"
+          readOnly: true
+      env:
+        - name: MAC_USER
+          valueFrom:
+            secretKeyRef:
+              name: $(params.macHostCredentials)
+              key: username
+        - name: MAC_HOST
+          valueFrom:
+            secretKeyRef:
+              name: $(params.macHostCredentials)
+              key: host
+        - name: KEYCHAIN_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: $(params.macSigningCredentials)
+              key: keychain_password
+        - name: SIGNING_IDENTITY
+          valueFrom:
+            secretKeyRef:
+              name: $(params.macSigningCredentials)
+              key: signing_identity
+        - name: APPLE_ID
+          valueFrom:
+            secretKeyRef:
+              name: $(params.macSigningCredentials)
+              key: apple_id
+        - name: TEAM_ID
+          valueFrom:
+            secretKeyRef:
+              name: $(params.macSigningCredentials)
+              key: team_id
+        - name: APP_SPECIFIC_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: $(params.macSigningCredentials)
+              key: app_specific_password
+        - name: QUAY_USER
+          valueFrom:
+            secretKeyRef:
+              name: $(params.quaySecret)
+              key: username
+        - name: QUAY_PASS
+          valueFrom:
+            secretKeyRef:
+              name: $(params.quaySecret)
+              key: password
+      script: |
+        #!/usr/bin/env bash
+        set -eux
 
-    # - name: sign-mac-binaries
-    #   image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
-    #   script: |
-    #     #!/usr/bin/env bash
-    #     # TODO CLDX-79
+        mkdir -p /root/.ssh
+        chmod 700 /root/.ssh
+        cp "/etc/secrets/mac_id_rsa" /root/.ssh/id_rsa
+        cp "/etc/secrets/mac_fingerprint" /root/.ssh/known_hosts
+        chmod 600 root/.ssh/known_hosts /root/.ssh/id_rsa
+
+        #TODO: cp SSH key to the container
+        # shell check complains about the variable (unsigned_digest) not being used but it is used in the script
+        # shellcheck disable=SC2034
+        unsigned_digest=$(cat "$(results.unsignedMacDigest.path)")
+        mac_signing_script="/tmp/mac_signing_script.sh"
+        set +x
```

Comment:
This is not needed (and the matching `set -x` after the EOF line). The contents of the here-document are not printed either way.